contrib/linearize/linearize-hashes.py

```diff
@@ -37,11 +37,11 @@ def __init__(self, host, port, username, password):
 	def execute(self, obj):
 		try:
 			self.conn.request('POST', '/', json.dumps(obj),
-				{ 'Authorization' : self.authhdr,
-				  'Content-type' : 'application/json' })
+				{'Authorization': self.authhdr,
+				'Content-type': 'application/json'})
 		except ConnectionRefusedError:
 			print('RPC connection refused. Check RPC settings and the server status.',
-			      file=sys.stderr)
+				file=sys.stderr)
```

Comment:
Same here and the hunk above. (No way we can force everyone to configure their editors to behave this way)